### PR TITLE
Don't enable event collection by default on OSP

### PR DESCRIPTION
**Pull request by leifmadsen**
Closes STF-1498

### CODE REVIEW

**Review by csibbitt — APPROVED** [review]
LGTM as a first pass

File: doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-director-operator-for-stf.adoc [hunk 1]
```diff
@@ -41,7 +41,7 @@ data:
 
         # enable Ceilometer metrics and events
         CeilometerQdrPublishMetrics: true
-        CeilometerQdrPublishEvents: true
+        CeilometerQdrPublishEvents: false
```
Comment (vkmc) [inline doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-director-operator-for-stf.adoc:44]: We could drop this config, the default value is false. See https://opendev.org/openstack/tripleo-heat-templates/src/branch/master/deployment/ceilometer/ceilometer-agent-notification-container-puppet.yaml#L53-L56

File: doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc [hunk 2]
```diff
@@ -28,9 +28,9 @@ parameter_defaults:
     ManagePolling: true
     ManagePipeline: true
 
-    # enable Ceilometer metrics and events
+    # enable Ceilometer metrics
     CeilometerQdrPublishMetrics: true
-    CeilometerQdrPublishEvents: true
+    CeilometerQdrPublishEvents: false
```
Comment (vkmc) [inline doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc:33]: Ditto

File: doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc [hunk 3]
```diff
@@ -104,9 +104,9 @@ parameter_defaults:
...
```
Comment (vkmc) [inline doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc:109]: Ditto

**Review by vkmc — APPROVED** [review]
Looks good to me. I think we could just delete the option from the environment file altogether and that would be enough, but if we want to be explicit with events not being enabled I would leave it as is.

File: doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-director-operator-for-stf.adoc (hunk 1 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-director-operator-for-stf.adoc:44]: OK thanks! I was looking at that but hadn't gotten around to looking up the defaults.

**Comment by leifmadsen** [comment]
I have updated everything after double checking the defaults Victoria pointed out along with the default Events publishing list (which is also empty). I've removed the extra items from the configuration now.

**Pull request merged**